Documentation/contributing/development/datapath_config.rst

```diff
@@ -0,0 +1,302 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _dpconfig:
+
+Configuring the Datapath
+========================
+
+Introduction
+~~~~~~~~~~~~
+
+In order for the Cilium datapath to function, it needs access to configuration
+data such as feature flags, addresses, timeouts, security IDs and all sorts of
+tunables and user configuration. These values are provided by the agent at the
+time of loading the bpf program. This page outlines the configuration mechanism,
```

Comment:
s/bpf/BPF/ ? Here and elsewhere

Reply:
Yep, this was in the back of my mind but forgot, thanks.